packages/sui-studio-utils/src/domain-builder/index.js

```diff
@@ -1,15 +1,18 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
Why? 🙄

Reply:
Because nesting ternaries are a warning.

Reply:
to avoid linting warnings

Reply:
I know, but why do we avoid the linting warning and do not take another approach in the code below?